invenio_vocabularies/config.py

```diff
@@ -42,8 +42,23 @@
         "validator": idutils.is_ror
     },
 }
+""""Generic identifier schemes, usable by other vocabularies."""
+
+VOCABULARIES_AFFILIATION_SCHEMES = {
+    **VOCABULARIES_IDENTIFIER_SCHEMES,
+}
 """Affiliations allowed identifier schemes."""
 
+VOCABULARIES_FUNDER_SCHEMES = {
+    **VOCABULARIES_IDENTIFIER_SCHEMES,
+}
+"""Funders allowed identifier schemes."""
+
+VOCABULARIES_AWARD_SCHEMES = {
+    **VOCABULARIES_IDENTIFIER_SCHEMES,
```

Comment:
minor: are we sure these are the same schemes? I see rdm-records had no list for starters... but I think for example "doi" is missing? @slint

Reply:
As discussed with @slint , `doi` and `url` are valid schemes for an award for now.